src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -288,7 +288,7 @@ Copyright (c) .NET Foundation. All rights reserved.
       </PropertyGroup>
     </When>
     
-    <When Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' and '$(IsExecutable)' == 'true'">
+    <When Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' and '$(HasRuntimeOutput)' == 'true'">
```

Comment:
Should stay '$(OutputType)' == 'Exe' or $(_IsExecutable).  I discussed this prevously that tests are not `dotnet run` able, becuase you can't just run the output assembly.